src/pulsar_sup.erl

```diff
@@ -22,7 +22,7 @@
 start_link() -> supervisor:start_link({local, ?MODULE}, ?MODULE, []).
 
 init([]) ->
-    ets:new(?PULSAR_METRICS_ETS, [named_table, public, {write_concurrency, true}]),
+    _ = ets:new(?PULSAR_METRICS_ETS, [named_table, public, {write_concurrency, true}]),
```

Comment:
use ordered set type.

Reply:
recently had to deal with the non-deterministic nature of 'set' 
made me thinking we should perhaps default to `ordered_set` instead.